docs-2.0-en/nebula-operator/8.custom-cluster-configurations/8.1.custom-conf-parameter.md

```diff
@@ -55,20 +56,124 @@ The following example uses a cluster named `nebula` and the cluster's configurat
         "auth_type": "password"
   ...
   ```
+
+  The types of parameters that can be added under the `config` field are as follows:
+  
+  - Parameters that support runtime dynamic modification. For information about the parameters that can be dynamically modified for each service, see the **Whether supports runtime dynamic modifications** column of the parameter tables in [Meta service configuration parameters](../../5.configurations-and-logs/1.configurations/2.meta-config.md), [Storage service configuration parameters](../../5.configurations-and-logs/1.configurations/4.storage-config.md), and [Graph service configuration parameters](../../5.configurations-and-logs/1.configurations/3.graph-config.md), respectively.
+
+  - Parameters that do not support runtime dynamic modification but are flagged when you run either of the following commands:
+    - `./bin/nebula-graphd --help`
+    - `curl -s  http://10.244.1.104:19669/flags`  
+
```

Comment:
I think we can focus more on the flagged fields rather than dynamically modificationable